packages/integration-sdk-private-test-utils/__fixtures__/synchronization-compressed/.j1-integration/graph/relationship-1.json

```diff
@@ -0,0 +1,2 @@
+%hι.�W�C�C4�[��?x�7�fEA�`6�+?
"�շX�P�-�
```

Comment:
In case you are wondering, this Google Translated is 
ᬥ Ũᰇ kyart ⺖ Ci 䏐 䌴 jjut  㽸 turbsyeob ፅ 䇺 Bi Fang 㼍 ⋒ hyaerb ᑘ Luan Ma ⷤਰ kkwik The Lai twap Long ߑ Nu  䬱 bwarh Cha edge jektae  chwet Yan Guo  Mulhouse Johnson ถ byaen Huan ssyeotseot 䳂  䠄ጡ ㅽ